lib/xrechnung.rb

```diff
@@ -287,7 +287,7 @@ def to_xml(indent: 2, target: "")
         "xmlns:cbc"          => "urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2",
         "xmlns:xsi"          => "http://www.w3.org/2001/XMLSchema-instance",
         "xsi:schemaLocation" => "urn:oasis:names:specification:ubl:schema:xsd:Invoice-2 http://docs.oasis-open.org/ubl/os-UBL-2.1/xsd/maindoc/UBL-Invoice-2.1.xsd" do
-        xml.cbc :CustomizationID, "urn:cen.eu:en16931:2017#compliant#urn:xoev-de:kosit:standard:xrechnung_3.0"
+        xml.cbc :CustomizationID, "urn:cen.eu:en16931:2017#compliant#urn:xoev-de:kosit:standard:xrechnung_2.2"
```

Comment:
@benedikt-voigt I don't know if we need to raise this version to 3.0 or not. if set to 3.0 the XSD validator doesn't work anymore so I removed the change. At first look I couldn't find an updated version for the XSD Validator but will have a second look tomorrow.

Reply:
If I read the documentation on [version 3.0.1](https://www.e-rechnung-bund.de/standard-xrechnung-3-0-1/) correctly then we would have to introduce 3 required fields to be compatible: BT-23, **BT-34** and BT-49.